doc/user/content/install.md

```diff
@@ -70,6 +70,16 @@ sudo apt update
 sudo apt install materialized
 ```
 
+### apt DEB822 (Ubuntu 16.04+, Debian 9+)
```

Comment:
16.04? It seems like support for the `Signed-By` PGP block was only added recently? https://blog.jak-linux.org/2021/06/20/migrating-away-apt-key/

Reply:
Oh, you're correct! I had a hard time to figure out when the deb822 support was added, but missed the `Signed-By`. It was added in [apt 2.3.10](https://salsa.debian.org/apt-team/apt/-/blob/main/debian/changelog#L182), which at this point is included in Debian bookworm/testing and Ubuntu Jammy (22.04). I better point this out!